src/envo/envo-edit.owl

```diff
@@ -27848,13 +27869,157 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000000> <http://purl.obolibrar
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000002> "http://orcid.org/0000-0002-3410-4655")
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000002> "https://en.wikipedia.org/wiki/N%C3%A9v%C3%A9")
-AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000002> "Firn derives from Névé that survives a full season of ablation.")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000002> "Firn derives from névé that survives a full season of ablation, which undergone some metamorphic transition.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000002> "firn"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000002> <http://purl.obolibrary.org/obo/ENVO_01000293>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000003> (nivation)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000003> "A material transformation process occurring due to a mass wasting process, or by cycles of freezing and thawing in which fresh snow is compacted into firn or névé.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000116> <http://purl.obolibrary.org/obo/ENVO_03000003> "Link to mass wasting process.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000003> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000003> "https://en.wikipedia.org/wiki/Nivation")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000003> "nivation"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000003> <http://purl.obolibrary.org/obo/ENVO_02500000>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000004> (glacial ice)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000004> "Any water ice that is part of a glacier.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000004> "http://orcid.org/0000-0002-3410-4655")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000004> "https://en.wikipedia.org/wiki/Glacier")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000004> "glacial ice"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000004> <http://purl.obolibrary.org/obo/ENVO_01000293>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000005> (erosionally enriched glacial ice)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000005> "Any glacial ice which contains material liberated during a glacial erosion process.")
```

Comment:
Avoid the usage of "Any", try:
```
Ice which is part of a glacier and contains material liberated during a glacial erosion process.
```

The hierarchy should look like this:

```
ice
- erosionally enriched ice
--- erosionally enriched glacial ice
```

Add the following subclass axiom to `erosionally enriched glacial ice`: 
```
'part of' some glacier
```

It will then be autoclassfied under `glacial ice` in the inferred hierarchy.